airbyte-webapp/.storybook/withProvider.tsx

```diff
@@ -38,11 +39,13 @@ export const withProviders = (getStory) => (
           <MemoryRouter>
             <IntlProvider messages={messages} locale={"en"}>
               <ThemeProvider theme={theme}>
-                <ConfigServiceProvider defaultConfig={defaultConfig} providers={[]}>
+                <ConfigServiceProvider config={config}>
```

Comment:
:information_source: This is kinda "funny" that this worked beforehand. So it seems that Storybook never did typecheck those files, until Storybook 7 (or use using a `main.ts` instead of `main.js`).